src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -71,12 +70,17 @@ function BulkEmailForm(props) {
    * @returns formatted Data
    */
   const formatDataForFormAction = (action) => {
+    const emailsIndividualLearners = emailLearnersList.map(({ email }) => email);
+    const extraTargets = { emails: emailsIndividualLearners };
+    const emailRecipients = editor.emailRecipients.filter((recipient) => recipient !== 'individual-learners-emails');
```

Comment:
how would we do this from the plugin? since the idea is not to modify the MFE

Reply:
In that case that won't be an option due to we would have to rewrite BulkEmailForm.js becuase all the logic is here and I don't think so that we have enough time for that